src/realm/unicode.cpp

```diff
@@ -92,6 +64,45 @@ std::wstring utf8_to_wstring(StringData str)
+    // clang-format on
```

Comment:
Problem is that it will flood the screen with useless data not meant to be read by a human...